### PR TITLE
Calendar module display improvements

### PR DESCRIPTION
**Pull request by ElviaBth**
🎩 What? Why?
Two issues came up during calendar testing on https://wedecide.green.ca/calendar.

- [x] Times on the calendar are displayed in 24 hour format even when system preference is for am/pm format. 

- [x] Responsive CSS framework is currently working for small screens, as per the 'month view'.

📌 Related Issues
 #176 on [gpc-decidim](https://github.com/Green-Party-of-Canada-Members/gpc-decidim/issues/176)

📷 Screenshots
  
Sets the height of the view

image
Get a am/pm format

♥️ Thank you!

### CODE REVIEW

File: app/presenters/decidim/calendar/event_presenter.rb [hunk 1]
```diff
@@ -99,7 +99,9 @@ def all_day?
       def hour
         return nil if start.is_a?(Date) || all_day?
 
-        start.strftime("%H:%M")
+        hour12 = Decidim::Calendar.calendar_options[:hour12]
+
+        hour12 ? start.strftime("%I:%M %p") : start.strftime("%H:%M")
```
Comment (microstudi) [inline app/presenters/decidim/calendar/event_presenter.rb:104]:
```suggestion
        format = Decidim::Calendar.calendar_options[:hour12] ? "%I:%M %p" : "%H:%M"

        start.strftime(format)
```
Maybe it's a little bit cleaner this way?

File: yarn.lock
Comment (microstudi) [inline yarn.lock:1]: We can completly remove this file

**Review by microstudi — COMMENTED** [review]
Nice work Elvia, let's do a bit of cleaning and that's it!

File: app/presenters/decidim/calendar/event_presenter.rb (hunk 1 above)
Comment (ElviaBth) [reply app/presenters/decidim/calendar/event_presenter.rb:104]: Thanks, done.

File: yarn.lock
Comment (ElviaBth) [reply yarn.lock:1]: Done

**Pull request merged**